scripts/cleanup_edomain_templates.py

```diff
@@ -12,7 +12,7 @@
 from threading import Lock, Thread
 
 from cfme.utils import net
-from cfme.utils.conf import cfme_data, credentials
+from cfme.utils.conf import cfme_data, credentials, provider_data as provider_yaml
```

Comment:
**QUESTION:** why as provider_yaml?

Reply:
ah good call - I had done this in a few places where provider_data was in use, not necessary here. but I would import as a different name to avoid using the same thing as a name already in use in the script.

Reply:
**SUGGESTION:** Add a comment explaining that.

It's all beautiful though.

Reply:
I dropped this, and have done a bit of logging refactor in the script, as well as just importing the whole conf module.